modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/idp/mgt/IdentityProviderMgtServiceTestCase.java

```diff
@@ -60,6 +61,10 @@
     private final String SAML2SSO_IDP_ENTITY_ID = "IdPEntityId";
 
     private static final String RANDOM_PASSWORD_GENERATED = "random-password-generated";
+    private final String invalidPageNumberErrorMessage
```

Comment:
let's make them private static final, since they are not changed in test methods.